sections/02_intro.md

```diff
@@ -16,12 +16,12 @@ demonstrated that a neural network could learn to identify cats simply by
 watching online videos [@doi:10.1109/ICASSP.2013.6639343]. Such approaches,
 termed deep learning, seem like a solution to the challenge presented by the
 growth of data in biomedicine. Perhaps these algorithms could identify the
-biological "cats" hidden in our data - the patterns unknown to researcher - and could act on them.
+biological "cats" hidden in our data - the patterns unknown to the researcher - and allow the researcher to act on them.
```

Comment:
Same here with biological. Please remove. Since it hits the same line, merging is going to result in this being re-added.